README.md

```diff
@@ -30,6 +30,7 @@ You'll also need to clone a new `.env` file from the `.env.template` to store lo
 
 ```bash
 $ cp .env.template .env  # (first time only)
+$ cp .env.template .env.test  # (first time only)
```

Comment:
This new step isn't actually required, because the `.env.test` will already exist when the project is cloned down, since it isn't git-ignored (unlike the `.env` file, which each dev needs to set up individually)